source/Mission.cpp

```diff
@@ -950,6 +951,10 @@ string Mission::BlockedMessage(const PlayerInfo &player)
 		out << "no additional space";
 	subs["<capacity>"] = out.str();
 
+	for(auto &keyValue : subs)
```

Comment:
```suggestion
	for(const auto &keyValue : subs)
```

coders are stupid sometimes, so dont give them the opportunity to be XD